tests/foreman/cli/test_import.py

```diff
@@ -299,37 +361,19 @@ def test_import_orgs_manifests(self, test_data):
         @assert: 3 Organizations are created with 3 manifests uploaded
 
         """
+        tmp_dir = self.default_dataset[0]
         files = dict(self.default_dataset[1])
-        files['users'] = update_csv_values(
-            files['users'],
-            u'organization_id',
-            test_data,
-            self.default_dataset[0]
-        )
-        csv_records = csv_to_dataset([files['users']])
```

Comment:
this chunk of code  has been replaced by the Import static method - `organization_with_tr_data_manifests`
as it's needed in multiple tests.